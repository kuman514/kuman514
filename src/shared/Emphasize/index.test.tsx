import { render } from '@testing-library/react';

import Emphasize from '.';

describe('Emphasize', () => {
  it('should show as expected', () => {
    const { container } = render(<Emphasize>Koishi Komeiji</Emphasize>);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should show as expected 2', () => {
    const { container } = render(
      <Emphasize customClassName="to-pink-300">Hoshino Takanashi</Emphasize>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
