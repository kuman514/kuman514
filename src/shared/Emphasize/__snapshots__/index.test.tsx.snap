// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Emphasize should show as expected 1`] = `
<strong
  class="bg-emphasize-underline from-transparent from-50% to-green-600 to-50% undefined"
>
  Koishi Komeiji
</strong>
`;

exports[`Emphasize should show as expected 2 1`] = `
<strong
  class="bg-emphasize-underline from-transparent from-50% to-green-600 to-50% to-pink-300"
>
  Hoshino Takanashi
</strong>
`;
